cmd/estuary-shuttle/main.go

```diff
@@ -223,22 +227,27 @@ func main() {
 		},
 		&cli.BoolFlag{
 			Name:  "logging",
+			Usage: "enable api endpoint logging",
 			Value: cfg.LoggingConfig.ApiEndpointLogging,
 		},
 		&cli.BoolFlag{
 			Name:  "write-log-flush",
+			Usage: "enable hard flushing blockstore",
 			Value: cfg.NodeConfig.HardFlushWriteLog,
 		},
 		&cli.BoolFlag{
 			Name:  "write-log-truncate",
+			Usage: "enables write log truncating",
 			Value: cfg.NodeConfig.WriteLogTruncate,
 		},
 		&cli.BoolFlag{
 			Name:  "no-blockstore-cache",
+			Usage: "disable blockstore caching",
 			Value: cfg.NodeConfig.NoBlockstoreCache,
 		},
 		&cli.BoolFlag{
 			Name:  "private",
+			Usage: "sets shuttle as private",
```

Comment:
More of a question: what does it mean for a shuttle to be private? Maybe we could make the wording of this explain what it is? If it's kind of a complex explanation nvm

Reply:
I honestly can not give another explanation besides making a shuttle private - other cannot connect to it or something. I added these simple comments as non where there in the first place